portal/templates/flask_user/login.html

```diff
@@ -1,5 +1,5 @@
 {% extends 'flask_user/public_base.html' %}
-
+{% block mainclass -%}class="login-form"{%- endblock %}
```

Comment:
Was just looking at the [whitespace stripping](http://jinja.pocoo.org/docs/dev/templates/#whitespace-control) with `-%}`. Will that have any effect since there's no whitespace to remove between the block tags?

Reply:
@ivan-c ivan, you are right, there won't be any effect.  thank you catching this. I was doing this out of habit.